### ISSUE TITLE
Disable advanced graphics features by default

### ISSUE THREAD

**Issue by BinToss** [HaloSPV3/HCE#50]
Part of #44

Only g-buffer and HUD visor should be on by default

**Comment by BinToss** [comment]
Working on it.
The changes so far are...

Decrement Default Settings

-- SPV3 POST-PROCESSING --
The following were changed to OFF by default:
^ Bloom
^ Lens Dirt
^ Dynamic Lens Flares (exclusive to SPV3.2)
^ Volumetric Lighting
^ FXAA (anti-aliasing)
^ Film Grain
^ Motion Blur
^ MXAO (MartMcFly's Ambient Obscurance)
^ DOF (Depth of Field)
^ SSR (Screen-Space Reflections)
^ Debanding

-- CHIMERA 2017 -- 
* Interpolation decremented from 9 to 8. I thought I changed that already...
* Anisotropic Filtering initialized to false. Technically, this is Halo's built-in setting, but Chimera can enforce it whereas Halo matches the setting to capable GPUs listed in its config.txt file...and doesn't auto-enable it when it can.
* BlockLOD bool initialized to false. Enabled == very expensive performance hit with small benefit to model quality at distances. I thought I fixed this already.
